src/main/kotlin/com/sourcegraph/cody/agent/protocol/AutocompleteParams.kt

```diff
@@ -1,13 +1,22 @@
 package com.sourcegraph.cody.agent.protocol
 
+import java.net.URI
+
 enum class AutocompleteTriggerKind(val value: String) {
   AUTOMATIC("Automatic"),
   INVOKE("Invoke"),
 }
 
 data class AutocompleteParams(
-    val filePath: String,
+    val uri: URI,
     val position: Position,
     val triggerKind: String? = AutocompleteTriggerKind.AUTOMATIC.value,
     val selectedCompletionInfo: SelectedCompletionInfo? = null
-)
+) {
+  constructor(
+      filePath: String,
+      position: Position,
+      triggerKind: String? = AutocompleteTriggerKind.AUTOMATIC.value,
+      selectedCompletionInfo: SelectedCompletionInfo? = null
+  ) : this(URI("file://$filePath"), position, triggerKind, selectedCompletionInfo)
```

Comment:
This URI constructor is going to crash in windows. It's better to go via nio.file.Path

Reply:
We already had the same issue before, which got fixed here https://sourcegraph.com/github.com/sourcegraph/jetbrains/-/commit/ac233c5959d5aff51ca4148a8d7cc5f883ac3581?visible=2

Reply:
Ah yes I remember now! Couldn't find it in slack or gh issue/PR search 🤔

Reply:
`type:diff` in Sourcegraph found 😜

Reply:
Done https://github.com/sourcegraph/jetbrains/pull/145/commits/9e2540ca0c88d35768e508a319d7f11f0432dfee